index.js

```diff
@@ -0,0 +1,21 @@
+const menuBar = document.querySelector('.menu-bar');
```

Comment:
**KINDLY WORK ON THESE CHANGES:**
_Provide the requested design:_
**_Yours:_**
![Screen Shot 2021-12-13 at 22 04 33](https://user-images.githubusercontent.com/22258274/145931185-0c841a58-6b7d-4be2-a0ed-7b384f460137.png)

**_Figma Design:_**
![Screen Shot 2021-12-13 at 22 05 22](https://user-images.githubusercontent.com/22258274/145931200-2c0a5f8b-7fe8-4bed-87d8-bfba8dbfdb35.png)

**_Fit your `menu` at the `body`._**